docs/how-to/store-tokens.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: Tokens
+title: Tokens: paying for storage
```

Comment:
This is what's currently causing the build to fail, per the logs: 

![image](https://user-images.githubusercontent.com/19171465/81849607-c71ab900-9524-11ea-8584-b2b3b6921fe0.png)

@har00ga Could you please check out the VuePress docs (or maybe just YAML docs) and see if there's a way to make it less confused about the double colons in the same line? 

We also need a capital P after the colon, even in sentence case, right?

Reply:
Ah sorry about that, should have tried the build before putting in the PR 🙂 No problem. I'll see if I can find a solution, if not we can just rephrase it to `Paying for storage (tokens)` or something, if that works? 

& ope, my bad! In British English (my natural writing) it's only capitalised if it's a proper noun, but in American English it is indeed always capitalised. Must have mixed that up -- I'll adjust it

Reply:
In most cases you'll want to change the title of an article in multiple places: 
- title in markdown (as you've done here)
- header in markdown
- Home component (eventually we'll make this pull from the same data the nav pulls from) at https://github.com/filecoin-project/filecoin-docs/blob/master/docs/.vuepress/theme/components/Home.vue

Is it intentional that you're changing this for the sidebar nav but not the homepage or page header?

Reply:
I generally leave top level `#` headers title capped (but none lower), though I suppose that's more a matter of preference. Happy to adjust them if you'd like

On the homepage component, seems I overlooked that. I'll modify those lines as well